### PR TITLE
fix: delete holiday visible in holidays list page

### PR DESCRIPTION
**Pull request by BLasan**
## Description
Remove deleted holidays from manage holidays list page.

## Related issues and discussion
#797 

## Screenshots, if any
webapp1

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
@karantakalkar Please review

File: src/app/organization/holidays/holidays.component.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ export class HolidaysComponent implements OnInit {
       this.holidaysData = [];
       this.organizationService.getHolidays(officeId).subscribe((holidays: any) => {
         this.holidaysData = holidays;
```
Comment (karantakalkar) [inline src/app/organization/holidays/holidays.component.ts:74]:
```suggestion
```

File: src/app/organization/holidays/holidays.component.ts [hunk 2]
```diff
@@ -72,6 +72,7 @@ export class HolidaysComponent implements OnInit {
...
+        this.holidaysData = this.holidaysData.filter((holiday: any) => holiday.status.value !== 'Deleted');
```
Comment (karantakalkar) [inline src/app/organization/holidays/holidays.component.ts:75]:
```suggestion
        this.holidaysData = holidays.filter((holiday: any) => holiday.status.value !== 'Deleted');
```

**Comment by BLasan** [comment]
@karantakalkar Done

**Pull request merged**